docs/modules/admin/pages/auto_suggest_search.adoc

```diff
@@ -1,32 +1,37 @@
-this suggestion. This feature is not turned on by default.  You must turn it on in the Administration module.
-
-
-== Enabling this Feature ==
-
-. To enable this feature, click *Administration* -> *Server Administration* -> *Global Flags*.
-. Scroll down to item 10, OPAC.
-. Double click anywhere in the row to edit the fields.
-. Check the box adjacent to *Enabled* to turn on the feature.
-. The *Value* field is optional.  If you checked *Enabled* in step 4, and you leave this field empty, then Evergreen will only suggest searches for which there are any corresponding MARC records.
-+
-NOTE: If you checked *Enabled* in step 4, and you enter the string, *opac_visible*, into this field, then Evergreen will suggest searches for which 
-there are matching MARC records with copies within your search scope.  For example, it will suggest MARC records with copies at your branch.
-+
-. Click *Save.*
-
-image::media/Auto_Suggest_in_Catalog_Search2.jpg[Auto_Suggest_in_Catalog_Search2]
-
-After turning on the feature, you may need to restart memcached and other services.
-
-== Using this Feature ==
-
-. Enter search terms into the basic search field.  Evergreen will automatically suggest search terms.
-. Select a suggestion to view records that are linked to this suggestion. 
-
-image::media/Auto_Suggest_in_Catalog_Search1.jpg[Auto_Suggest_in_Catalog_Search1]
-
+= Auto Suggest in Catalog Search =
+:toc:
+
+The auto suggest feature suggestions for completing search terms as the user enters his search query.  Ten suggestions are the default, but the number of suggestions is configurable at 
+the database level. Scroll through suggestions with your mouse, or use the arrow keys to scroll through the suggestions.  Select a suggestion to view records that are linked to 
+this suggestion. This feature is not turned on by default.  You must turn it on in the Administration module.
+
+
+== Enabling this Feature ==
+
+. To enable this feature, click *Administration* -> *Server Administration* -> *Global Flags*.
+. Click *Filter* under *Name*.  Choose *Is Exactly* and search 'opac.use_autosuggest'. 
+. Double click anywhere in the row to edit the fields.
+. Check the box adjacent to *Enabled* to turn on the feature.
++
+NOTE: If you checked *Enabled* in step 4, and it should autofill *opac_visible* under *Value*. Evergreen will suggest searches for which 
+there are matching MARC records with copies within your search scope.  For example, it will suggest MARC records with copies at your branch.
++
+. Click *Save.*
+
+image::autosuggest/global_flag_autosuggest.png[Global Flag Setting for Auto Suggest]
+
+After turning on the feature, it's recommended to:
+
+. Restart or reload Apache on all webservers.
+. Restart memcached.  You can use this command:
+----
+memcdump --servers localhost|grep open-ils.search | xargs -n 1 memcrm --servers localhost
+----

```

Comment:
Noting a couple issues with the suggested documentation change:

- the command is not the one to restart memcached, but then again, it's better to avoid directing users to restart memcached unless necessary. The effect of the command is to instead clear certain values from the cache.
- A better command to target the keys that actually affect autosuggest would be `memcdump --servers localhost | grep ^EGWeb | xargs -n 1 memcrm --servers localhost`

Reply:
I updated this with the suggestion.  Thank you!